common/src/main/java/whocraft/tardis_refined/common/capability/TardisLevelOperator.java

```diff
@@ -250,15 +252,17 @@ public void setDoorClosed(boolean closeDoor) {
             TardisEvents.DOOR_OPENED_EVENT.invoker().onDoorOpen(this);
         }
 
-        if (extManager != null) {
-            if (extManager.getLastKnownLocation() != null) {
-                extManager.setDoorClosed(closeDoor);
+
+
+        if (this.pilotingManager != null) {
```

Comment:
getCurrentLocation() used multiple times, make it a local variable

Reply:
It is used once in this instance.